tests/vnc_test.go

```diff
@@ -240,6 +242,21 @@ var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][lev
 				return nil
 			}, 60*time.Second).ShouldNot(HaveOccurred())
 		})
+
+		It("should allow creating a VNC screenshot in PNG format", func() {
+			filePath := filepath.Join(GinkgoT().TempDir(), "screenshot.png")
+
+			cmd := clientcmd.NewVirtctlCommand("vnc", "screenshot", "--namespace", vmi.Namespace, "--file", filePath, vmi.Name)
+			Expect(cmd.Execute()).To(Succeed())
+
+			f, err := os.Open(filePath)
+			Expect(err).ToNot(HaveOccurred())
+
+			img, err := png.Decode(f)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(img.Bounds().Size().X).To(BeNumerically("==", 720))
+			Expect(img.Bounds().Size().Y).To(BeNumerically("==", 400))
```

Comment:
Just to confirm: the framebuffer size 720x400 is specific to that particular VM image that is run in the test, right?

Reply:
It is the default resolution of the qemu boot screen. Since the VM we start has no actual VM image, it should be stable to use, because there boots no OS which can change it.